config/shared-config/config.yaml

```diff
@@ -0,0 +1,9 @@
+sharedSecretFile: $(CONFIG_MOUNT_PATH)/shared-secret
+baseUrl: $(OAUTH_HOST)
+serviceProviders:
+- type: GitHub
+  clientId: "123"
```

Comment:
kustomize isn't really made for templating. This would need to be handled by copying the kustomize config to a temp dir, running envsubst on it and then applying. I opted for not doing that at this point in time because this config needs manual update after deploying to the cluster anyway.

Reply:
ok